src/wazuh_db/wdb_integrity.c

```diff
@@ -111,6 +111,9 @@ void wdbi_report_removed(const char* agent_id, wdb_component_t component, sqlite
                 cJSON_AddItemToObject(j_data, "board_serial", cJSON_CreateString((const char*) sqlite3_column_text(stmt, 0)));
                 router_handle = router_inventory_events_handle;
                 break;
+            case WDB_SYSCOLLECTOR_NETADDRESS:
+                cJSON_AddStringToObject(j_msg_to_send, "action", "deleteNetworkAddress");
+                cJSON_AddItemToObject(j_data, "item_id", cJSON_CreateString((const char*) sqlite3_column_text(stmt, 0)));
```

Comment:
You must set the handle and add a `break`

Reply:
Thanks!

Reply:
Check all cases